feconf.py

```diff
@@ -635,6 +635,17 @@ def get_empty_ratings():
 # questions.
 MAX_NUMBER_OF_SKILL_IDS = 20
 
+# The maximum number of blog post cards to be visible on each page in blog
+# homepage.
+MAX_LIMIT_FOR_CARDS_ON_HOMEPAGE_PAGE = 10
+
+# The maximum number of blog post cards to be visible on each page in blog
+# author's page.
+MAX_LIMIT_FOR_CARDS_ON_BLOG_AUTHORS_PAGE = 12
```

Comment:
Let's stop calling this the "blog author's page". It's not the blog author's page -- the page does not belong to the blog author. It's a list of all blog posts that are written by a particular author.

Perhaps the "author-specific blog posts page" or something like that.

Reply:
Done.

Reply:
This is not fixed. Please do not say something is "done" when it is not fixed.

Reply:
Fixed. Sorry.